apphub/anomaly_detection/alocc/alocc_torch.py

```
@@ -176,6 +178,12 @@ def get_estimator(epochs=20, batch_size=128, train_steps_per_epoch=None, save_di
 
     x_eval, y_eval = np.concatenate([x_eval0, x_eval1]), np.concatenate([y_eval0, y_eval1])
     eval_data = fe.dataset.NumpyDataset({"x": x_eval, "y": y_eval})
+    return train_data, eval_data
+
+
+def get_estimator(epochs=20, batch_size=128, train_steps_per_epoch=None, save_dir=tempfile.mkdtemp()):
```

Ruff (0.11.9)

Do not perform function call `tempfile.mkdtemp` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)